packages/odyssey/src/scss/abstracts/_mixins.scss

```diff
@@ -14,13 +14,40 @@
   position: absolute;
   width: 1px;
   height: 1px;
-  margin: -1px;
-  padding: 0;
+  margin-block: -1px;
+  margin-inline: -1px;
+  padding-block: 0;
+  padding-inline: 0;
   overflow: hidden;
   clip: rect(0 0 0 0);
   border: 0;
 }
 
+/* stylelint-disable liberty/use-logical-spec */
+
+@mixin border-radius( $all, $top-right: $all, $bottom-right: $all, $bottom-left: $all, $top-left: $all ) {
+  border-start-start-radius: $top-left;
+  border-start-end-radius: $top-right;
+  border-end-end-radius: $bottom-right;
+  border-end-start-radius: $bottom-left;
+
+  @supports not (border-end-end-radius: 1px) {
```

Comment:
First I've seen `@supports` - this is interesting. Is this something we can use broadly across different browser targets? I would think if it isn't supported in say Safari or IE11 than the following declaration block would be ignored, in that case we would want to invert the logic here to apply the explicit LTR and RTL styles, and then use `@supports` to target logical properties yea?

Reply:
By using `not` here, we ensure that the fallback styles aren't competing in cases where both are syntaxes supported. Otherwise, we end up relying on property order to determine whether logical or physical properties should take precedence.

When we view FF and Chrome in the inspector, they only pick up the logical properties:

<img width="505" alt="Screen Shot 2021-06-21 at 1 55 52 PM" src="https://user-images.githubusercontent.com/36284167/122826792-6b2e7200-d298-11eb-8132-1da492e727f5.png">

<img width="586" alt="Screen Shot 2021-06-21 at 1 50 54 PM" src="https://user-images.githubusercontent.com/36284167/122826805-6e296280-d298-11eb-9f9b-6088c2fccb77.png">

In Safari, we see that the fallbacks are picked up and the additional `rtl` selectors are present, while the unsupported syntax is ignored.

<img width="313" alt="Screen Shot 2021-06-21 at 1 53 24 PM" src="https://user-images.githubusercontent.com/36284167/122826890-88fbd700-d298-11eb-86fb-3d9b5cdfc79b.png">